torch/quantization/fx/quantize.py

```diff
@@ -565,6 +566,8 @@ def prepare_dynamic(self, model, qconfig_dict, inplace=False):
 
     def convert(self, observed, inplace=False, debug=False):
         assert self.activation_post_process_map is not None
+        # move to cpu since we only have quantized cpu kernels
+        observed.eval().cpu()
```

Comment:
thoughts on if we can let the user do this instead, and assert that the model is on CPU (or handle GPU)?  I think doing it here might be surprising to users